src/decision/dMCDA.jl

```diff
@@ -469,49 +482,61 @@ Tuple (SH, wsh)
     5. high cover (weights importance of sites with high cover of coral to shade)
 """
 function create_shade_matrix(A::Matrix{Float64},
-    max_area::Vector{Float64},
+    k_area::Vector{T},
     wt_conn_shade::T,
     wt_waves::T,
     wt_heat::T,
     wt_predec_shade::T,
     wt_predec_zones_shade::T,
-    wt_hi_cover)::Tuple{Matrix{Float64}, Vector{Float64}} where {T<:Float64}
+    wt_hi_cover,
+)::Tuple{Matrix{Float64},Vector{Float64}} where {T<:Float64}
+
+    # Define weights vector
+    wsh = [
+        wt_conn_shade,
+        wt_conn_shade,
+        wt_waves,
+        wt_heat,
+        wt_predec_shade,
+        wt_predec_zones_shade,
+        wt_hi_cover,
+    ]
+
     # Set up decision matrix to be same size as A
     SH = copy(A)
 
     # Remove consideration of site depth as shading not accounted for in bleaching model yet
-    SH = SH[:, 1:end-1]
+    SH = SH[:, 1:(end - 1)]
 
-    wsh = [wt_conn_shade, wt_conn_shade, wt_waves, wt_heat, wt_predec_shade, wt_predec_zones_shade, wt_hi_cover]
-    wsh .= mcda_normalize(wsh)
+    # Get area of coral cover (m²) by subtracting leftover space from total area for coral.
+    SH[:, 8] = k_area .- SH[:, 8]
```

Comment:
Looking at this again, why not just pass in the absolute coral cover (which is already calculated elsewhere) rather than redoing the calculation again?

Reply:
What is absolute cover called in scenario now? I can see `loc_k_area`, which is the k area, `C_t`, `C_Cover` and `loc_coral_cover `which are all forms of relative cover. I tried searching for variables with `m\^2` , or `abs` in them but couldn't find anything so I used `leftover_space_m2` instead

Reply:
Ah okay, so we don't have it already calculated then. Okay to ignore this comment then, thank you.